mkosi/resources/mkosi-initrd/mkosi.conf

```diff
@@ -90,8 +99,12 @@ KernelModulesInclude=
                     /snd-intel-dspcfg.ko
                     /snd-soc-hda-codec.ko
                     /squashfs.ko
+                    /thinkpad_acpi.ko
```

Comment:
This one pulls in quite a bit of extra stuff and is rather hardware specific, can you double check you actually need this one in the initrd?

Reply:
The other modules all look good to me to add

Reply:
I added a bunch to get usb to work, but yeah it works without this one, dropped for now